crates/solver/src/settlement_submission.rs

```diff
@@ -130,7 +133,7 @@ impl TransactionStrategy {
             TransactionStrategy::Eden(args) => Some(args),
             TransactionStrategy::Flashbots(args) => Some(args),
             TransactionStrategy::PublicMempool(args) => Some(args),
-            TransactionStrategy::DryRun => None,
+            _ => None,
```

Comment:
nit: Personally I would avoid `_` here and just name all the variants.
I like being saved by compiler when I miss to add a new case...